service/service.go

```diff
@@ -56,17 +56,30 @@ func Run(ctx context.Context, cfg *config.Config, serviceList *ExternalServiceLi
 		return nil, err
 	}
 
+	// Get Kafka producer for reindex task counts
+	producerForReindexTaskCounts, err := serviceList.GetKafkaProducerForReindexTaskCounts(ctx, cfg)
```

Comment:
So we need to extend the registered checkers to include new kafka producers and arguably Zebedee health check

And also update the graceful shutdown to stop writing to the kafka topics (close them down)

Reply:
Fixed.